libmamba/include/mamba/core/util.hpp

```diff
@@ -26,6 +27,14 @@ namespace mamba
 {
     class Context;
 
+    namespace
```

Comment:
We should avoid anonymous namespaces in headers, because all the anonymous namespaces of a compilation unit are collapsed into a single one, and this could lead to names clash.

Reply:
Actually we may want to encapsulate these variables in functions now that they are to be used in different places and exposed in the API.

Reply:
Maybe just functions returning their values?

Reply:
I meant functions encapsulating the definition, yes